src/javascript/JContent/JContent.accordion-items.jsx

```diff
@@ -18,6 +18,9 @@ import {Sql2SearchQueryHandler} from '~/JContent/ContentRoute/ContentLayout/quer
 
 const filesRegex = /^\/sites\/[^/]+\/files((\/.*)|$)/;
 const contentsRegex = /^\/sites\/[^/]+\/contents((\/.*)|$)/;
+const contentFolderRegex = /^\/sites\/[^/]+\/contents.*/;
+const folderRegex = /^\/sites\/[^/]+\/files.*/;
+const everythingUnderSitesRegex = /^\/sites\/.*/;
```

Comment:
The 2 first regex are actually matching /files and /contents , exactly like the contentFolderRegex and folderRegex (but the 2 latter also matches /filesxx and /contentsxx)

```suggestion
const filesRegex = /^\/sites\/[^/]+\/files\/.*/;
const contentsRegex = /^\/sites\/[^/]+\/contents\/.*/;
const contentFolderRegex = /^\/sites\/[^/]+\/contents((\/.*)|$)/;
const folderRegex = /^\/sites\/[^/]+\/files((\/.*)|$)/;
const everythingUnderSitesRegex = /^\/sites\/.*/;
```